Apps/W1/ReviewGLEntries/app/src/pages/ReviewedGLEntries.Page.al

```diff
@@ -0,0 +1,55 @@
+namespace Microsoft.Finance.GeneralLedger.Review;
+
+using Microsoft.Finance.GeneralLedger.Account;
+using Microsoft.Finance.GeneralLedger.Ledger;
+page 22208 "Reviewed G/L Entries"
+{
+    Caption = 'Reviewed G/L Entries';
+    PageType = List;
+    DataCaptionExpression = GetCaption();
+    ApplicationArea = Basic, Suite;
+    DeleteAllowed = false;
+    InsertAllowed = false;
+    ModifyAllowed = false;
+    Editable = false;
+    Permissions = tabledata "G/L Entry" = rimd,
```

Comment:
Is there any reason for those `imd` permission?
And for the indirect setup permissions?